src/components/icons/bus.js

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
Helst ingen tomme linjer på starten eller slutten av fila. Du kan sette ESLint-regelen `"no-multiple-empty-lines": ["error", { "max": 2, "maxBOF": 0, "maxEOF": 0 }],` for å autofikse sånt.

Reply:
Det kan vi få fikset 👍